index.bs

```diff
@@ -1066,6 +1066,43 @@ If the attribute identifies a URL that is auxilliary to the element's purpose, l
 
 Note: remember that attributes containing URLs should be represented in IDL as {{USVString}}; see [[#idl-string-types]].
 
+<h3 id="html-overloading">Give each HTML element a single purpose</h3>
+
+Each HTML element should have a clear purpose.
+Using HTML attributes can modify the semantics of an element,
+but attributes should not fundamentally alter that purpose.
+
+Rather than defining elements that can operate in different modes,
+having a separate element for each mode is preferable.
```

Comment:
Counterexample: `<legend>`, `<caption>`, and `<figcaption>` probably should have all been the same element (but can't be for compat reasons).